web/src/util.ts

```diff
@@ -533,3 +534,12 @@ export function compare_a_b<T>(a: T, b: T): number {
     }
     return -1;
 }
+
+export function get_final_topic_display_name(topic_display_name: string): string {
+    if (topic_display_name === "") {
+        // TODO: Add translation logic here when implementing
+        // `realm_empty_topic_display_name` as a realm setting.
+        return realm.realm_empty_topic_display_name;
```

Comment:
I think we need translation logic here for the feature we're building now? I think correct code should involve:

```
if (realm.realm_empty_topic_display_name === "general chat") {
   return $t({defaultMessage: "general chat"})
}
```

Reply:
Made the changes + added a unit test for this function.

I was planning to do this when we'll provide a way to configure `realm_empty_topic_display_name` to admins or changing "test general chat" to "general chat". But, yeah, no harm in doing that now.